src/main/java/hudson/plugins/warnings/AggregatedWarningsResult.java

```diff
@@ -31,9 +32,30 @@
      *            the parsed result with all annotations
      * @param defaultEncoding
      *            the default encoding to be used when reading and parsing files
+     *
+     * @deprecated see {@link #AggregatedWarningsResult(Run, BuildHistory, ParserResult, String)}
```

Comment:
NIT: just for consistency, sometimes you use 'see' sometimes you say 'use' later in the code.